python/xorbits/_mars/dataframe/core.py

```diff
@@ -2183,7 +2183,9 @@ def __repr__(self):
         return self._to_str(representation=True)
 
     def __mars_tensor__(self, dtype=None, order="K"):
-        return self.to_tensor().astype(dtype=dtype, order=order, copy=False)
+        tensor = self.to_tensor()
+        dtype = dtype if dtype is not None else tensor.dtype
```

Comment:
If `dtype` is None, do we need to call `astype`?

Reply:
The order could be different.